dev/com.ibm.ws.concurrent.mp_fat/fat/src/com/ibm/ws/concurrent/mp/fat/MPConcurrentTxTest.java

```diff
@@ -53,7 +62,9 @@ public static void tearDown() throws Exception {
                    "java.lang.IllegalStateException", // attempt to use same transaction on 2 threads at once
                    "javax.transaction.xa.XAException" // transaction marked rollback-only due to intentionally caused error
     })
+
     @Test
+    @SkipForRepeat(MPContextProp11RepeatAction.ID)
```

Comment:
@njr-11 This test fails with version 1.1, when you review please decide if a follow up change is needed to fix that.

Reply:
This worked fine for me when I ran it locally with 1.1.  Tom is going to follow up and see if he can reproduce and report what the error is.  This shouldn't hold up delivery of this pull because it's a question of whether this additional test can be enabled, rather than of a potentially failing test being delivered.